docs/DeveloperGuide.md

```diff
@@ -278,8 +278,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | Teaching Assistant                                        | add students to TAPA                                                                   | know which students are taking my classes                                                     |
 | `* * *`  | Teaching Assistant                                        | delete a specific student from TAPA                                                    | remove student who has withdrawn from the module during the semester                          |
 | `* * *`  | Teaching Assistant                                        | list students alphabetically                                                           | print students in a list as their names would appear on the official class list               |
-| `* * *`  | Teaching Assistant                                        | save existing data to hard drive                                                       | access the data again when I reopen the app                                                   |
+| `* * *`  | Teaching Assistant                                        | tag students                                                                           | know what modules they are taking                                                             |
+| `* * *`  | Teaching Assistant                                        | filter students by their tags                                                          | view the progress of students in each separate tag (different class/module)                   |
+| `* * *`  | Teaching Assistant                                        | define my own tasks for each module                                                    | customise the tasks to each module                                                            |
+| `* * *`  | Teaching Assistant                                        | assign tasks to multiple students                                                      | allocate tasks to students without having to individually assign them                         |
```

Comment:
Tbh it feels a little weird for "assign tasks to multiple students" to have a higher priority than "assign tasks to specific students".
Maybe we should readjust the priority level for either/both of these user stories?

Reply:
Alright, I swapped the priorities for these 2 user stories.